crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -509,6 +618,8 @@ pub enum TypedTransaction {
     EIP2930(Signed<TxEip2930>),
     /// EIP-1559 transaction
     EIP1559(Signed<TxEip1559>),
+    /// EIP-4844 transaction
+    EIP4844(Signed<TxEip4844Variant>),
     /// op-stack deposit transaction
     Deposit(DepositTransaction),
```

Comment:
same here i take that we redefine the enum vs reusing from alloy bc of the deposit tx right?

@mattsse do u have any ideas for doing this with a super-enum?

something like `enum TypedTransaction { #[flatten] Inner(alloy::TypedTransaction), Deposit(...)`?